.github/workflows/pr_push.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -y cmake clang-format-15
+          sudo apt-get install -y cmake clang-format-15 libhwloc-dev
```

Comment:
hmm... why did it work previously (without this package)?

Reply:
It did not.